pydis_site/apps/content/resources/guides/pydis-guides/contributing/installing-project-dependencies.md

```diff
@@ -39,3 +39,27 @@ $ poetry run task precommit
    ![PyCharm Poetry Environment](/static/images/content/contributing/pycharm_poetry.png)<br/>
 6. PyCharm will automatically install the packages required into a virtual environment.<br/>
    ![PyCharm Project Interpreter](/static/images/content/contributing/pycharm_poetry_success.png)
+
+
+## Installing specific dependency groups
+
+In some cases, like when you'll be using the configuration bootstrapping script, you might need to install extra dependencies that are
+not necessary for the bot project to function.
```

Comment:
That's no longer true right? should this section still exist?

Reply:
yes yes, forgot about that one, deleted